app/assets/stylesheets/components/_select-with-search.scss

```diff
@@ -295,7 +295,8 @@
   @media (min-width: 640px) {
     .choices__list--dropdown .choices__item--selectable,
     .choices__list[aria-expanded] .choices__item--selectable {
-      padding-right: 100px;
+      padding-right: 5px;
+      word-break: break-word;
```

Comment:
The CSS rule `word-break: break-word;` [is deprecated](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break), so we shouldn't really be using it.

However, I think probably what we're trying to achieve here is to restore the browser's default line wrap behaviour. So instead of setting it here, I think we should try removing the following rules:

https://github.com/alphagov/whitehall/blob/2bb0c25121226c2aea98d5087a5288182ca10163/app/assets/stylesheets/components/_select-with-search.scss#L215

https://github.com/alphagov/whitehall/blob/2bb0c25121226c2aea98d5087a5288182ca10163/app/assets/stylesheets/components/_select-with-search.scss#L249

Which should mean we don't need this CSS rule at all.

Reply:
Good spot! That fixed it. Thanks for the review 👍